src/components/networked.js

```diff
@@ -54,6 +58,22 @@ AFRAME.registerComponent('networked', {
     document.body.dispatchEvent(this.entityCreatedEvent());
   },
 
+  takeOwnership: function() {
+    const owner = this.data.owner;
+    const lastOwnerTime = this.lastOwnerTime;
+    const now = NAF.utils.now();
+    if(owner && !this.isMine() && lastOwnerTime < now) {
+      networkedEntity.setAttribute("networked", { owner: NAF.clientId });
+      this.lastOwnerTime = now;
+      return true;
+    }
+    return false;
+  }
+
+  update: function(oldData) {
+    this.syncAll();
```

Comment:
This feels a bit heavy handed, id imagine this results in sending a bunch more sync packets during init since update will get called as owner and networkId are set. I assume this is being done to handle owner changing?

We had discussed making owner no longer a component property, since its invalid to directly set it, so why not just call `syncAll` inside `takeOwnership`